Docs/Book/Cookbook.md

```diff
@@ -409,6 +409,8 @@ The code:
     fp = numpy.zeros((1,))
     DataStructs.ConvertToNumpyArray(AllChem.GetMorganFingerprintAsBitVect(m5, 2), fp)
 
+    # Reshape to a matrix for one sample
```

Comment:
I close this pull request and submit a new one with your solution. That PR will also contain changes to the print statements to work with Python 2 and 3.